### PR TITLE
Refactor signals

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Summary

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

File: lib/sanbase/signals/trigger/settings/result_builder/result_builder.ex [hunk 1]
```diff
@@ -0,0 +1,45 @@
+defmodule Sanbase.Signal.ResultBuilder do
```
Comment (tspenov) [inline lib/sanbase/signals/trigger/settings/result_builder/result_builder.ex]: we have this file twice

File: lib/sanbase/signals/trigger/settings/result_builder/transformer.ex [hunk 2]
```diff
@@ -0,0 +1,34 @@
+defmodule Sanbase.Signal.ResultBuilder.Transformer do
```
Comment (tspenov) [inline lib/sanbase/signals/trigger/settings/result_builder/transformer.ex]: Also we have this one twice

File: lib/sanbase/signals/trigger/settings/result_builder/result_builder.ex (hunk 1 above)
Comment (IvanIvanoff) [reply lib/sanbase/signals/trigger/settings/result_builder/result_builder.ex]: We do not - one is the result builder for the signal triggering, one is the result builder in the history dir
`Sanbase.Signal.**History**.ResultBuilder.Transformer`

File: lib/sanbase/signals/trigger/settings/result_builder/transformer.ex (hunk 2 above)
Comment (IvanIvanoff) [reply lib/sanbase/signals/trigger/settings/result_builder/transformer.ex]: Same as the answer above - the other one is in the history dir and is used for building the result for historical activity

Comment (IvanIvanoff) [reply lib/sanbase/signals/trigger/settings/result_builder/transformer.ex]: The content is the same in the PR - I'm reworking this locally and will push the changes soon

File: lib/sanbase/signals/trigger/settings/result_builder/result_builder.ex (hunk 1 above)
Comment (tspenov) [reply lib/sanbase/signals/trigger/settings/result_builder/result_builder.ex]: ah, I see

File: lib/sanbase/signals/history/result_builder/transformer.ex [hunk 3]
```diff
@@ -0,0 +1,21 @@
+defmodule Sanbase.Signal.History.ResultBuilder.Transformer do
```
Comment (tspenov) [inline lib/sanbase/signals/history/result_builder/transformer.ex]: Is this module used ?

Comment (IvanIvanoff) [reply lib/sanbase/signals/history/result_builder/transformer.ex]: https://github.com/santiment/sanbase2/pull/1478/files#diff-c8093b1bd1638fe9f4801b96f805f9c3R3

Comment (tspenov) [reply lib/sanbase/signals/history/result_builder/transformer.ex]: Ctrl-e, Ctrl-g doesn't show on collapsed file :(

**Pull request merged**